### DISCUSSION TITLE
Unable to install headers

### DISCUSSION THREAD

**Discussion by oj-philz** [Cyfrin/security-and-auditing-full-course-s23#64] [Q&A]
i have installed both rust and cargo and i have cloned the transmissions headers repository but when i run the commad
```
cargo install --path .
```
I get the errors below;
```
error: could not compile `headers` (bin "headers") due to previous error
error: failed to compile `headers v0.1.0 (/home/ojphilz/Desktop/Cyfrin Updraft - Security and Auditing course/3-passwordstore-audit/headers)`, intermediate artifacts can be found at `/home/ojphilz/Desktop/Cyfrin Updraft - Security and Auditing course/3-passwordstore-audit/headers/target`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

**Comment by cromewar** [comment]
hello @oj-philz 
Seems you might not have Rust properly installed, please check if the Rust installation works properly and it was added to your path. Based on your error message seems the cargo compiler path is missing and can be corrected adding it to path like this;

```
export CARGO_TARGET_DIR=/path/to/your/target/dir
```

Or by re installing Rust.

**Comment by PatrickAlphaC** [comment]
Going to close this due to stale. Let us know if you still need help!
